### ISSUE TITLE
Remote browser for SauceLabs doesn't work for Watir 6.6.2

### ISSUE THREAD

**Issue by drodriguezaltimetrik** [watir/watir#614]
If you select :remote and pass the options object, you get a `undefined method 'delete' for nil:NilClass`.
The error is found inside `Watir::Capabilities` class

```
def initialize(browser, options = {})
 @browser = browser == :remote ? @options.delete(:browser).to_sym : browser.to_sym
 @selenium_browser = browser == :remote || options[:url] ? :remote : browser

 @options = options.dup
 @selenium_opts = {}
end
```
Note the first line that calls to `@options` that's not created yet. It should call to `options`
Also, when using SauceLabs, you don't pass the `:browser` hash since all the needed info is inside `:desired_capabilities`. If you do a `options.delete(:browser).to_sym` will show you a `undefined method 'to_sym' for nil:NilClass` error

**Comment by titusfortner** [comment]
yup, I see the problem. fixing it now.

**Comment by titusfortner** [comment]
Fixed in 6.6.3
Thanks for raising the issue.

**Comment by titusfortner** [comment]
That being said. With Watir 6.6, you don't need to use `:remote` or initialize any Selenium classes to start a browser on Sauce. http://watir.com/watir-6-6/#watir-capabilities

Obviously if I'd written the correct test for it in the first place to ensure backward compatibility, though... :)
